examples/web-chat/src/Message.ts

```diff
@@ -1,4 +1,4 @@
-import { DecodedMessage } from "@waku/core/lib/message/version_0";
+import { IDecodedMessage } from "@waku/interfaces";
```

Comment:
Developer should not use `IDecodedMessage` but should use the `DecodedMessage` related to the decoder they use.

`IDecodedMessage` is useful for implementers of `Decoder`.

Reply:
But here it is used as a type definition which it interchangeable with `DecodedMessage` and reduces risk of having unwanted js in the bundle (shouldn't be the case though)

Reply:
> But here it is used as a type definition which it interchangeable with DecodedMessage and reduces risk of having unwanted js in the bundle (shouldn't be the case though)

As mentioned here: https://github.com/waku-org/js-waku-examples/pull/214#discussion_r1119559227 I think we should not want library consumers to use `@waku/interfaces` as it contains way more than what they need. Lots of types are implementation purposes only.

In term of your concerns, IMO the solution would seem to be:

```
import type { DecodedMessage } from "@waku/core";
```

Assuming `DecodedMessage` is export from `@waku/core` root index file.